interview-questions.md

```diff
@@ -4,62 +4,57 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What does CRUD stand for?
 
-  Your answer: create, read, update, delete
+  Your answer:
 
-  Researched answer: In computer programming, create, read, update, and delete (CRUD) are the four basic functions of persistent storage.
+  Researched answer:
 
 
 
 2. What are the 5 HTTP verbs?
 
-  Your answer: I would google 5 HTTP verbs
+  Your answer:
 
   Researched answer:
-The primary or most-commonly-used HTTP verbs (or methods, as they are properly called) are POST, GET, PUT, PATCH, and DELETE.
+
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to render "Hello World" on the page?
 
-  Your answer: import class Components, extend components, render, return, and export.
```

Comment:
this question was a little confusing and your answer is great. but just so you know, we were looking for: import, class name extends, render, return, div/React.Fragment would be a bonus, export included in this answer somehow